public/config.js

```diff
@@ -712,8 +712,28 @@ const CONFIG = {
           template: 'https://www.bing.com/search?q={@}'
         }
       }
+    },
+    brave: {
+      name: 'Brave',
+      bgColor: {
+        type: 'gradient',
+        gradientType: 'linear',
+        colors: ['#FB542B', '#343546']
+      },
+      textColor: '#e8e8e8',
+      types: {
+        query: {
+          template: 'https://search.brave.com/search?q={$}'
+        },
+        calculator: {
+          template: 'https://search.brave.com/search?q={$}'
```

Comment:
You should use the origin query instead of the final one (`@` symbol instead of `$`) because it will search for the answer of a math expression instead an expression itself